glide.lock

```diff
@@ -203,7 +204,7 @@ imports:
   - lib/selector/tokenizer
   - lib/set
   - lib/testutils
-  - lib/validator
+  - lib/validator/v3
```

Comment:
Think you need to run `make update-vendor` again, looks like it's pulled in the v3 package.  (Probably doesn't have any effect, but seems wrong.)